console/src/components/Common/Icons/Download.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
As we have a new Icon component in `UIKit/` we will get rid of `Common/Icon` dir. Can you add those icons to `UIKit/`?